rules/how-to-monetize-apps/rule.md

```diff
@@ -14,66 +16,87 @@ redirects:
+* Your product is a AAA gaming title
+* Your product is a useful, but not essential, productivity tool
+* Your product is *NOT* a SaaS product (one-off payments are difficult to reconcile against ongoing cloud and hosting costs)
 
 ::: good
-Good example: free apps with in-app purchases  
-:::
-
-
-
-
-::: bad
-Bad example: paid with currency  
+![Figure: The Legend of Engadine on iOS and Android is free to play, but a one-time purchase unlocks premium features.](engadine.png)
+**Good Example:** [The Legend of Engadine](https://engadine.net) on iOS and Android is free to play, but a one-time purchase unlocks premium features.
 :::
+    
+One-time fees work with either paid apps or freemium apps; users can either buy the app up-front or use the app for free, and pay to unlock premium features.
 
+### Subscription fee
+Subscriptions have become much more popular, even for traditional desktop software. Rather than pay a one-time fee, users pay a recurring monthly or annual fee to use the software. This might be the right approach for you if:
+* Your product is a SaaS product
+* Your product is likely to get recurring regular use *without* you needing to continually drive engagement (e.g., an essential, or at least high value, productivity tool)
+* Your product has other ongoing maintenance costs (e.g. content development)
 
 ::: good
-Good example: paid with abstract currency  
+![Figure: Microsoft 365 is available to home and business users for a low recurring cost.](microsoft365.png)
+**Good Example:** [Microsoft 365](https://www.microsoft.com/microsoft-365) (formerly Office 365) is a suite of productivity tools that run on mobile, desktop and web. For a low recurring cost, users get access to these apps and the connected services that tie them together in the cloud.
 :::
 
-
-
+### Micro-transactions
+Micro-transactions is the practice of continually charging comparatively small amounts of money over a long period of time. Generally, micro-transactions are considered unethical because they are closely aligned with nefarious practices like [dark patterns](https://www.youtube.com/watch?v=lJUW0iZzAaQ), 'pay-to-win' or exploiting gambling addition.
+    
+For these reasons, micro-transactions are [increasingly](https://www.vg247.com/australian-government-rules-games-microtransactions-must-address-packaging) [subject](https://www.gamespot.com/articles/you-might-have-to-prove-your-age-to-buy-a-loot-box/1100-6474443/) to scrutiny from [governments](https://www.aph.gov.au/Parliamentary_Business/Committees/House/Social_Policy_and_Legal_Affairs/Onlineageverification/Report) and [regulators](https://www.gamesindustry.biz/australian-investigation-finds-loot-boxes-are-pscyholgocially-akin-to-gambling). This might be the right approach for you if:
+* You don't mind profiting from the suffering of others
+* You are morally bankrupt
+* You would consider a sacrifice of your first-born child to the Devil in exchange for wealth to be a worthwhile transaction
```

Comment:
Test fail on these bullet-points.

Reply:
These were my favorite part of the rule!!

Reply:
lol yeah these are funny but can't go in the rule

Reply:
Done.